src/components/views/settings/tabs/room/BridgeSettingsTab.js

```diff
@@ -0,0 +1,171 @@
+/*
+Copyright 2019 New Vector Ltd
```

Comment:
We assign all copyright to the foundation:
```suggestion
Copyright 2019 The Matrix.org Foundation C.I.C.
```